druid/src/window.rs

```diff
@@ -426,11 +426,8 @@ impl<T: Data> Window<T> {
         for &r in invalid.rects() {
             piet.clear(
                 Some(r),
-                if self.transparent {
-                    Color::TRANSPARENT
-                } else {
-                    env.get(crate::theme::WINDOW_BACKGROUND_COLOR)
-                },
+                env.try_get(crate::theme::WINDOW_BACKGROUND_COLOR)
+                    .unwrap_or_else(|_| self.background.clone()),
```

Comment:
now the background should drawn in backends?

Reply:
outside of the scope. this PR is about ease of use and allowing semitransparent backgrounds. lets stick to that issue.

Reply:
beyond scope creep, I'm not comfortable doing that change personally ATM. due to it impacting all the backends and it isn't just a manual translation of logic. it'll blow up testing, debugging, and implementation work and personally I only need semitransparent windows. best to hold off until someone is in the backends for actual work. I will be doing a wayland backend in the future if no one beats me to it. I'll investigate actually painting the background at that time.

Reply:
p.s. thanks for being patient with me. only my 3rd week of programming in rust. still working my way through the ecosystem and learning its idiosyncrasies.

Reply:
makes sense